tests/fixtures/test_bad_privacy_notices_update.yml

```diff
@@ -0,0 +1,34 @@
+privacy_notices:
```

Comment:
These and similar files are added here to test various scenarios when templates are added on startup